app/GonimoBack.hs

```diff
@@ -22,26 +21,24 @@ import           Gonimo.Server.Effects.Development
 #else
 import           Gonimo.Server.Effects.Production 
 #endif
-import qualified Gonimo.Server.State as Server
-import           Gonimo.WebAPI
 
 logHandle :: Handle
 logHandle = stderr
 
 main :: IO ()
 main = do
   let subscriberPath = "subscriber"
-  subscriber <- atomically $ makeSubscriber subscriberPath runStderrLoggingT 
+  subscriber' <- atomically $ makeSubscriber subscriberPath runStderrLoggingT
   pool       <- runLoggingT (createSqlitePool "testdb" 1) doLogging
   families   <- newTVarIO Map.empty
   flip runSqlPool pool $ runMigration migrateAll
   let config = Config {
     configPool = pool
   , configLog  = logToHandle logHandle
-  , state      = Server.State families 
-  , subscriber = subscriber
+  , state      = families
```

Comment:
That's actually likely to change - that the online state only consists of the family map. But we will leave it like this for now - this should really be easy to change later on, especially if we use lenses.